kdumpctl

```diff
@@ -16,6 +16,7 @@ KDUMP_INITRD=""
 TARGET_INITRD=""
 #kdump shall be the default dump mode
 DEFAULT_DUMP_MODE="kdump"
+KDUMP_STATUS="/var/crash/kdump.status"
```

Comment:
The /var/crash could not exist, how to ensure it being created?   Is this status saved on dump target or the crashing machine?  Probably save on crashing machine looks better